sample_factory/algo/learning/learner.py

```diff
@@ -111,13 +111,30 @@ def invoke_after_each_epoch(self):
         return True
 
 
+class LinearDecayScheduler(LearningRateScheduler):
+    def __init__(self, cfg):
+        num_updates = cfg.train_for_env_steps // cfg.batch_size * cfg.num_epochs
+        self.linear_decay = LinearDecay([(0, cfg.learning_rate), (num_updates, 0)])
```

Comment:
Should we decay to 0 or to some really small value like 1e-6?
What do the other frameworks do?